### ISSUE TITLE
A Notification System for Telegram

### ISSUE THREAD

**Issue by dendisuhubdy** [ericaltendorf/plotman#163]
Hi there @ericaltendorf! Cool work!

I'd like to develop an additional notification system for current plots based on `plotman interactive` for a telegram bot for a telegram group. Is this of interest to you/the community?

**Comment by ericaltendorf** [comment]
Could we make it generic?  E.g., as discussed in https://github.com/ericaltendorf/plotman/issues/40 ?  Also tagging @altendky for opinions.

**Comment by altendky** [comment]
For what it is worth, my next priority is to hopefully setup test coverage monitoring and work on filling out the tests.  Really, every change we make should have testing to go along with it.  So, I'm unlikely to be focusing on designing and implementing a general and complete notification system 'soon'.  That said, having a couple subprocess calls at interesting points is a thing that seems reasonable to do.  Certainly a generic system of hooks ought to have that as a feature so it wouldn't add too much in the way of backwards compatibility concerns.

**Comment by BasilHorowt** [comment]
I agree that [general hooks](https://github.com/ericaltendorf/plotman/issues/40) might be the best approach for this kind of request.  

Perhaps that means we should close this specific issue?
